### DISCUSSION TITLE
How to create a new folder and a task within?

### DISCUSSION THREAD

**Discussion by matthew-mayer** [dahall/TaskScheduler#899] [Q&A] [answered]
Hi,

I was wondering if someone can show me an example of how to create a new folder and then save a new task within?

Thanks,

Matt

**Comment by dahall** [comment, marked as answer]
```cs
TaskDefinition td = TaskService.Instance.NewTask();
// Add definition info as in online samples

TaskFolder newFld = TaskService.Instance.RootFolder.CreateFolder("NewFolder");
newFld.RegisterTaskDefinition("MyTask", td);
```

**Comment by matthew-mayer** [comment]
Awesome, thank you so much! Is there any way to do this in the Fluent
syntax?

On Wed, Sep 22, 2021 at 6:48 PM David Hall ***@***.***> wrote:

> TaskDefinition td = TaskService.Instance.NewTask();// Add definition info as in online samples
> TaskFolder newFld = TaskService.Instance.RootFolder.CreateFolder("NewFolder");newFld.RegisterTaskDefinition("MyTask", td);
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dahall/TaskScheduler/discussions/899#discussioncomment-1372422>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVYA33PHWIVZR4JBVWZGA7TUDKBPNANCNFSM5ESPKI4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
*Matt Mayer | **IT **Help desk*

*Bay Photo Lab **| ***@***.*** ***@***.***>*

*920 Disc Dr, Scotts Valley, CA 95066*

*(831) 475-6886 x179*

**Comment by dahall** [comment]
I just looked and I failed to add that to the Fluent classes. I'll mark this as an enhancement request.
